app/tests/integration/nimbus/test_e2e_create_experiment.py

```diff
@@ -8,50 +8,55 @@
 from selenium.common.exceptions import NoSuchElementException, TimeoutException
 
 
-@pytest.mark.nondestructive
-def test_create_new_experiment(selenium, base_url):
-    experiment_name = "test_create_new_experiment"
-
-    selenium.get(base_url)
-    home = HomePage(selenium, base_url).wait_for_page_to_load()
-    experiment = home.create_new_button()
-    experiment.public_name = experiment_name
-    experiment.hypothesis = "smart stuff here"
-    experiment.application = "DESKTOP"
+def create_experiment(selenium, home_page, data):
+    experiment = home_page.create_new_button()
+    experiment.public_name = data.public_name
+    experiment.hypothesis = data.hypothesis
+    experiment.application = data.application
 
     # Fill Overview Page
     overview = experiment.save_and_continue()
-    overview.public_description = "description stuff"
+    overview.public_description = data.public_description
     overview.select_risk_brand_false()
     overview.select_risk_revenue_false()
     overview.select_risk_partner_false()
 
     # Fill Branches page
     branches = overview.save_and_continue()
     branches.remove_branch()
-    branches.reference_branch_name = "name 1"
-    branches.reference_branch_description = "a nice experiment"
-    branches.feature_config = "No Feature Firefox Desktop"
+    branches.reference_branch_name = data.branches[0].name
+    branches.reference_branch_description = data.branches[0].description
+    branches.feature_config = data.branches[0].config
 
     # Fill Metrics page
     metrics = branches.save_and_continue()
 
     # Fill Audience page
     audience = metrics.save_and_continue()
-    audience.channel = "Nightly"
-    audience.min_version = 80
-    audience.targeting = "TARGETING_MAC_ONLY"
-    audience.percentage = 50.0
-    audience.expected_clients = 50
+    audience.channel = data.audience.channel
+    audience.min_version = data.audience.min_version
+    audience.targeting = data.audience.targeting
+    audience.percentage = data.audience.percentage
+    audience.expected_clients = data.audience.expected_clients
     audience.save_btn()
-    audience.save_and_continue()
+    review = audience.save_and_continue()
 
     # Review
     selenium.find_element_by_css_selector("#PageSummary")
+    return review
 
 
-def test_create_new_experiment_remote_settings(selenium, base_url):
-    experiment_name = "test_create_new_experiment_remote_settings"
+@pytest.mark.nondestructive
+def test_create_new_experiment(selenium, base_url, default_data):
+    default_data.public_name = "test_create_new_experiment"
```

Comment:
Love this

Reply:
What's being passed in here?  An instance of a data class?  Does mutating it share data between tests?